server/src/lib.rs

```diff
@@ -13,63 +13,64 @@ use std::time::Duration;
 const TARGET_FOOD_COUNT: usize = 600;
 const MINIMUM_SAFE_MASS_RATIO: f32 = 0.85;
 
-#[spacetimedb(table(public))]
+#[spacetimedb::table(name = config, public)]
 pub struct Config {
-    #[primarykey]
+    #[primary_key]
     pub id: u32,
     pub world_size: u64,
 }
 
-#[spacetimedb(table(public))]
+#[spacetimedb::table(name = entity, public)]
 pub struct Entity {
-    #[autoinc]
-    #[primarykey]
+    #[auto_inc]
+    #[primary_key]
     pub id: u32,
     pub position: Vector2,
     pub mass: u32,
 }
 
-#[spacetimedb(table(public))]
-#[spacetimedb(index(btree, name = "player_id_index", player_id))]
+#[spacetimedb::table(name = circle, public)]
 pub struct Circle {
-    #[primarykey]
+    #[primary_key]
     pub entity_id: u32,
+    #[index(btree)]
     pub player_id: u32,
     pub direction: Vector2,
     pub magnitude: f32,
     pub last_split_time: Timestamp,
 }
 
-#[spacetimedb(table(public))]
+#[spacetimedb::table(name = player, public)]
 pub struct Player {
-    #[primarykey]
+    #[primary_key]
     identity: Identity,
     #[unique]
-    #[autoinc]
+    #[auto_inc]
     player_id: u32,
     name: String,
 }
 
-#[spacetimedb(table(public))]
+#[spacetimedb::table(name = logged_out_player, public)]
 pub struct LoggedOutPlayer {
```

Comment:
@cloutiertyler shall we use the new table feature to factor this type out?

Reply:
Yes, but not right now.